bugbug/calibration_wrapper.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from sklearn.isotonic import IsotonicRegression
+from sklearn.model_selection import train_test_split
+
+
+# Wrapper class to calibrate a model
+class IsotonicRegressionCalibrator:
+    def __init__(self, model_class):
+        self.model_class = model_class
+        self.model = None
+        self.calibrated = False
+        self.X_train, self.X_val, self.X_test = None, None, None
+        self.y_train, self.y_val, self.y_test = None, None, None
+        self.ir = None
+
+    def fit(self, X, y):
+        self.X_train, self.X_test, self.y_train, self.y_test = train_test_split(
+            X, y, test_size=0.2, random_state=42
+        )
+        self.X_train, self.X_val, self.y_train, self.y_val = train_test_split(
+            self.X_train, self.y_train, test_size=0.2, random_state=42
+        )
```

Comment:
Wasn't I supposed to split the data into three parts? because we would be doing the calibration the validation set! 
Also i dont get wydm by this "What you get in the fit() method is always the training set."

Reply:
The data is already splitted into train/test before being passed to the `fit()` method. So all the data that we have here is training data. We need to split it into train/val.